features/admin.login-flow.ai.v1/components/login-flow-ai-banner.tsx

```diff
@@ -198,7 +198,7 @@ const LoginFlowAIBanner: FunctionComponent<IdentifiableComponentInterface> = (
                                 link={ getLink("develop.applications.editApplication.common.signInMethod." +
                                     "conditionalAuthenticaion.ai.learnMore") }
                             >
-                                <Trans i18nKey={ "extensions:common.learnMore" }>
+                                <Trans i18nKey={ "console:common.learnMore" }>
```

Comment:
IMO we should use `common:learnMore` in these places.

Reply:
This PR is to move the console specific i18n from extensions to i18n module, hence the console prefix. Given the current time, I think we can address this as an improvement later.

Reply:
My concern is we will miss this redundant content. WDYT about;
1. keeping these as a separate namespace and properly cleanup later?
2. Or replacing `extensions:common.learnMore` with `common:learnMore`?

Reply:
Anyway, we can assume we'll clean up the console namespace someday.

Reply:
`replacing extensions:common.learnMore with common:learnMore?` -> +1 to this. The only problem is I'm unable to do this improvement at this time due to time constraints. I will create a separate issue to cleanup the console namespace.

Reply:
Fixed for learn more string in [c62ac96](https://github.com/wso2/identity-apps/pull/7235/commits/c62ac96eae501dff20685abfe77d0ed285602cba)